tests/exchanges/test_bybit.py

```diff
@@ -28,19 +29,26 @@ def test_get_name(bybit_exchange):
 @pytest.mark.asyncio
 async def test_spot_orders_parameters(bybit_exchange):
     exchange = exchanges.Bybit(bybit_exchange)
-    await create_order_tests.create_order_mocked_test_args(
-        exchange,
-        exchange_private_post_order_method_name="privateLinearPostOrderCreate",
-        exchange_request_referral_key=exchange.HEADER_SPOT_KEY,
-        should_contains=False)
+    await exchange._exchange.connector.client.load_markets()
+    exchange._exchange.connector.client.market("BTC/USDT:USDT")['spot'] = True
```

Comment:
we should test "BTC/USDT" as well since we are forcing a ccxt version without ":XYZ" symbols